src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/Providers/AbstractSemanticQuickInfoProvider.cs

```diff
@@ -274,8 +283,18 @@ internal abstract partial class AbstractSemanticQuickInfoProvider : AbstractQuic
             var semanticModel = await document.GetSemanticModelForNodeAsync(token.Parent, cancellationToken).ConfigureAwait(false);
             var enclosingType = semanticModel.GetEnclosingNamedType(token.SpanStart, cancellationToken);
 
-            var symbols = semanticModel.GetSemanticInfo(token, document.Project.Solution.Workspace, cancellationToken)
-                                       .GetSymbols(includeType: true);
+            var syntaxFacts = document.Project.LanguageServices.GetService<ISyntaxFactsService>();
+
+            ImmutableArray<ISymbol> symbols;
+            if (syntaxFacts.IdentifiesLambda(token))
```

Comment:
yeah. i would just have IdentifiesLambda just pulled into this type as an abstract method.

Reply:
Done :-)